### DISCUSSION TITLE
求音乐源

### DISCUSSION THREAD

**Discussion by HVGNc** [lyswhut/lx-music-desktop#1880] [General]
求音乐源  多谢兄弟姐妹了。求可用音乐源

**Comment by EDSTimmy** [comment]
六音音源（可以试试看）：）

**Reply by yuzhufeng** [reply]
在哪

**Comment by qq8179417** [comment]
https://liuyang-public.oss-cn-beijing.aliyuncs.com/sixyin-music-source-v1.2.0.js
https://liuyang-public.oss-cn-beijing.aliyuncs.com/flower-v1.0.0.js
https://raw.niuma666bet.buzz/Huibq/keep-alive/master/render_api.js

**Reply by qq8179417** [reply]
分享3个哈

**Reply by Rei203** [reply]
怎么用啊

**Reply by nuyiy** [reply]
感谢

**Reply by kilighzw** [reply]
感谢

**Reply by ccqaiqd** [reply]
感谢

**Reply by fgr178707** [reply]
> 怎么用啊

在线导入

**Comment by qq8179417** [comment]
我给你的是链接&nbsp; &nbsp;复制粘贴到音乐源面板
raw.niuma666bet.buzz/Huibq/keep-alive/master/render_api.js


liuyang-public.oss-cn-beijing.aliyuncs.com/sixyin-music-source-v1.2.0.js


liuyang-public.oss-cn-beijing.aliyuncs.com/flower-v1.0.0.js





&nbsp;


.
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年12月30日(星期一) 上午9:36
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [lyswhut/lx-music-desktop] 求音乐源 (Discussion #1880)





 
感谢
 
&mdash;

**Reply by qq8179417** [reply]
①：https://ghproxy.net/https://raw.githubusercontent.com/pdone/lx-music-source/main/sixyin/latest.js
②：https://ghproxy.net/https://raw.githubusercontent.com/pdone/lx-music-source/main/huibq/latest.js
③：https://ghproxy.net/https://raw.githubusercontent.com/pdone/lx-music-source/main/flower/latest.js
④：https://tt.tenmeng.com/moonue/js/yecao202412.js
⑤：https://raw.niuma666bet.buzz/Huibq/keep-alive/master/render_api.js
